src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -176,7 +176,7 @@
 
   <!-- This node represents the ReadOnlySpan<T> for the elements passed as the last argument to a collection builder
        method. It does not survive lowering. -->
-  <Node Name="BoundCollectionBuilderElementsPlaceholder" Base="BoundValuePlaceholderBase" />
+  <Node Name="BoundCollectionBuilderElementsPlaceholder" Base="BoundValuePlaceholderBase" DoesNotSurvive="LocalRewriting"  />
```

Comment:
nit: the comment on line above ("It does not survive lowering.") could be removed